bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/auxfile/AuxFileEditor.scala

```diff
@@ -27,7 +27,7 @@ object AuxFileEditor {
       lab.text = size.toString
   })
 
-  private val Format = new SimpleDateFormat("MM/dd/yy HH:mm:ss") {
+  private val Format = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss") {
```

Comment:
The columns in the "File Attachment" tab look fine:

<img width="1033" alt="file_attachment" src="https://user-images.githubusercontent.com/215438/26836232-4aa5a91c-4aa8-11e7-8b05-2e4bf1e8f397.png">

However it doesn't look good in the "Sync History" tab:

<img width="1034" alt="sync_history" src="https://user-images.githubusercontent.com/215438/26836280-6d49b71a-4aa8-11e7-944c-829dd0221aee.png">

Do you think it's worth trying to increase the column width for *start* and *end*? I have no idea how difficult or easy that would be.